nixos/modules/services/hardware/amdgpu.nix

```diff
@@ -0,0 +1,40 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.amdgpu;
+in {
+  options.hardware.amdgpu = {
+    legacySupport.enable = lib.mkEnableOption ''support for old AMD graphics cards, this uses
+      `amdgpu` instead of `radeonsi` driver Southern Islands (SI) and Sea Islands (CIK) cards
```

Comment:
Wording nit: missing "for" between "driver" and "Southern Islands". Also, this should probably have some rough idea of what the affected hardware actually is (HD 7000/8000 series), and warn the users that analog video outputs will not work (which is why it's not enabled by default in the first place).

Reply:
Done.